### PR TITLE
Feature/duplicatecard

### PR DESCRIPTION
**Pull request by RandyGitProjects**
Created duplicate cards using javascript which in turn makes dryer code for the HTML cards section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,20 +36,7 @@ <h4>Body</h4>
...
-        <div class="body-section">
-          <h3 class="idea-title">Idea title</h3>
-          <p class="idea-body">Idea body. Don't ever play yourself. Every chance I get, I water the plants, Lion! Cloth talk.</p>
-        </div>
-        <div class="bottom-bar">
-          <button class="comment-btn" type="submit"><img src="assets/comment.svg" alt="comment button"></button>
-          <label for="comment-btn">Comment</label>
-        </div>
-      </section>
+      <!-- New cards are inserted here -->
```
Comment (Denzel-Turambi) [inline index.html:39]: Nice job at making the HTML more dry.

File: main.js [hunk 2]
```diff
@@ -12,16 +12,31 @@ var ideaBody = document.querySelector(".idea-body");
...
 var formSearch = document.querySelector("#search-form");
-
+var ideaCards = document.querySelector(".idea-cards");
 //event listeners:
 saveButton.addEventListener("click", saveIdea);
 
 //functions:
+
+
+
```
Comment (Denzel-Turambi) [inline main.js:22]: We can delete these lines here.

File: main.js [hunk 3]
```diff
@@ -12,16 +12,31 @@ var ideaBody = document.querySelector(".idea-body");
...
 function saveIdea(event) {
     event.preventDefault();
     var newCard = new Ideas(formTitle.value, formBody.value);
-    ideaTitle.innerText = newCard.title;
-    ideaBody.innerText = newCard.body;
+    ideaCards.innerHTML += `<section class="card">
```
Comment (Denzel-Turambi) [inline main.js:26]: Could possibly start a new line after `+=`.

File: main.js [hunk 4]
```diff
@@ -12,16 +12,31 @@ var ideaBody = document.querySelector(".idea-body");
...
+    </div>
+    <div class="body-section">
+      <h3 class="idea-title">${formTitle.value}</h3>
+      <p class="idea-body">${formBody.value}</p>
+    </div>
+    <div class="bottom-bar">
+      <button class="comment-btn" type="submit"><img src="assets/comment.svg" alt="comment button"></button>
+      <label for="comment-btn">Comment</label>
+    </div>
+  </section>`
```
Comment (asutherland91) [inline main.js:39]: This looks excellent! But I'm wondering if it actually shouldn't maybe be a different function?

**Review by brendel-r — APPROVED** [review]
Great job! Although, I do agree with Adriane. That is something we should think about!!

**Pull request merged**